applications/GeoMechanicsApplication/custom_workflows/dgeosettlement.cpp

```diff
@@ -331,7 +331,6 @@ std::shared_ptr<StrategyWrapper> KratosGeoSettlement::MakeStrategyWrapper(const
         rProjectParameters["solver_settings"], GetComputationalModelPart());
     KRATOS_ERROR_IF_NOT(solving_strategy) << "No solving strategy was created!" << std::endl;
 
-    solving_strategy->Initialize();
```

Comment:
In the end, this was the culprit, the too early initialization (before the processes were initialized).